stable/dex/templates/deployment.yaml

```
@@ -37,6 +37,8 @@ spec:
       serviceAccountName: {{ template "dex.serviceAccountName" . }}
       nodeSelector:
 {{ toYaml .Values.nodeSelector | indent 10 }}
+      tolerations:
```

I know this is not done this way for the other ones, but the code would be more resilient with :
```suggestion
{{- with .Values.tolerations }}
      tolerations:
{{ toYaml . | indent 8 }}
{{- end }}
```